backend/src/org/commcare/xml/FixtureXmlParser.java

```diff
@@ -50,11 +50,14 @@ public FormInstance parse() throws InvalidStructureException, IOException,
 
         String userId = parser.getAttributeValue(null, "user_id");
 
-        //Get to the data root
-        parser.nextTag();
+        TreeElement root;
+        if (nextTagInBlock("fixture")) {
+            //TODO: We need to overwrite any matching records here.
+            root = new TreeElementParser(parser, 0, fixtureId).parse();
+        } else {
+            root = new TreeElement();
```

Comment:
Oh, do we need one? If we have an empty fixture, I don't think we need to add a record to our DB. I think an empty <fixture/> tag should either delete an existing fixture or be ignored entirely. We can't really make a meaningful "Base" element like this